### ISSUE TITLE
With the string only keys for session in 0.6.0 how should assigns be accessed in templates?

### ISSUE THREAD

**Issue by foobarto** [phoenixframework/phoenix_live_view#595]
As per topic. Assigned variables to session with atoms as keys were available in templates using `@my_var` syntax. However with 0.6.0 session assigns have to use string keys leading to following exception:

```
assign @my_var not available in eex template.

Please make sure all proper assigns have been set. If this
is a child template, ensure assigns are given explicitly by
the parent template as they are not automatically forwarded.

Available assigns: [:live_view_module, :socket, :view_module, :view_template, "_csrf_token", "my_var"]
```

The workaround for now for me is to access it from `@socket.assigns["my_var"]` however this is a bit verbose compared to what we could do before.
Is there a better way?

**Comment by aptinio** [comment]
The session now only accepts string keys (see [`Router.live/3`](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.Router.html#live/3) and [`LiveView.Controller.live_render/3`](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.Controller.html#live_render/3)), but socket assigns still use atoms as keys.

**Comment by chrismccord** [comment]
Can you show the `live_render` call you are performing? `@socket.assigns["my_var"]` is definitely not something you need to do, but it's hard to say what you need without knowing more.

**Comment by foobarto** [comment]
Thanks you for replying. I figured it out eventually what I was doing wrong. I used `mount/3` as a simple pass through for session data. The obvious in hindsight solution is to simply remap session values from string keys to atom based keys in `mount/3` 

So now when 

```elixir
live_render(conn, MyApp.MyLiveView, session: %{"my_var" => "something"})
```
The solution is to write `mount/3` as following
```elixir
def mount(_params, session, socket) do
  {:ok, assign(socket, my_var: session["my_var"])}
```
and now `@my_var` can be used in the template.
